app/problem/views.py

```diff
@@ -0,0 +1,37 @@
+""" Endpoint for create problem"""
+from typing import Tuple
+from flask import Blueprint, jsonify, request, Response
+from app.problem.file_creator import FileCreator
+
+problem = Blueprint("problem", __name__)
+
+
+@problem.route("/problem", methods=["POST"])
+def judge_new_problem() -> Tuple[Response, int]:
```

Comment:
```suggestion
def new_problem() -> Tuple[Response, int]:
```

Solo es una sugerencía, no es la mayor cosa. Es que al leer judge_new_problem pienso que se va a juzgar

Reply:
Resuelto en https://github.com/MinervaLMS/judge-api/commit/27f61ec3a0457bd808e1ae888c8ee58f019c350d